src/script/components/MessagesList/Message/ContentMessage/asset/ImageAsset.tsx

```diff
@@ -87,8 +88,16 @@ const ImageAsset: React.FC<ImageAssetProps> = ({asset, message, onClick, teamSta
     username: `${message.user().name()}`,
   });
 
+  const style: CSSObject = {
+    aspectRatio: `${asset.ratio}`,
```

Comment:
It can't be like that?

```suggestion
    aspectRatio: asset.ratio,
```

Reply:
No, it's a number and I need a string 🤷

Reply:
Ahh.. sure! :)